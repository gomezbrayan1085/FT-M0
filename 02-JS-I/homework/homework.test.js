const homework = require("./homework")
// @ponicode
describe("homework.suma", () => {
    test("0", () => {
        let result = homework.suma(100, 410)
        expect(result).toMatchSnapshot()
    })

    test("1", () => {
        let result = homework.suma(30, 50)
        expect(result).toMatchSnapshot()
    })

    test("2", () => {
        let result = homework.suma(320, 1)
        expect(result).toMatchSnapshot()
    })

    test("3", () => {
        let result = homework.suma(320, 30)
        expect(result).toMatchSnapshot()
    })

    test("4", () => {
        let result = homework.suma(90, 50)
        expect(result).toMatchSnapshot()
    })

    test("5", () => {
        let result = homework.suma(NaN, NaN)
        expect(result).toMatchSnapshot()
    })
})

// @ponicode
describe("homework.resta", () => {
    test("0", () => {
        let result = homework.resta(100, 100)
        expect(result).toMatchSnapshot()
    })

    test("1", () => {
        let result = homework.resta(320, 90)
        expect(result).toMatchSnapshot()
    })

    test("2", () => {
        let result = homework.resta(550, 520)
        expect(result).toMatchSnapshot()
    })

    test("3", () => {
        let result = homework.resta(350, 4)
        expect(result).toMatchSnapshot()
    })

    test("4", () => {
        let result = homework.resta(350, 90)
        expect(result).toMatchSnapshot()
    })

    test("5", () => {
        let result = homework.resta(-Infinity, -Infinity)
        expect(result).toMatchSnapshot()
    })
})

// @ponicode
describe("homework.multiplica", () => {
    test("0", () => {
        let result = homework.multiplica(400, 1)
        expect(result).toBe(undefined)
    })

    test("1", () => {
        homework.multiplica(100, 4)
    })

    test("2", () => {
        homework.multiplica(1, 380)
    })
})
