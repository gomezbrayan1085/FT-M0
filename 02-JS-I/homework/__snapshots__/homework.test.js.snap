// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homework.resta 0 1`] = `0`;

exports[`homework.resta 1 1`] = `230`;

exports[`homework.resta 2 1`] = `30`;

exports[`homework.resta 3 1`] = `346`;

exports[`homework.resta 4 1`] = `260`;

exports[`homework.resta 5 1`] = `NaN`;

exports[`homework.suma 0 1`] = `510`;

exports[`homework.suma 1 1`] = `80`;

exports[`homework.suma 2 1`] = `321`;

exports[`homework.suma 3 1`] = `350`;

exports[`homework.suma 4 1`] = `140`;

exports[`homework.suma 5 1`] = `NaN`;
